frontend/src/components/CampoDeArquivo.vue

```diff
@@ -130,71 +187,110 @@ async function enviarArquivo(e) {
       class="campo-de-arquivo__chamada-pendente horizontal"
     />
 
-    <div
-      v-else
-      class="campo-de-arquivo__area-de-envio flex center g1 pt1 pb1"
-      tabindex="0"
-      :class="{ 'campo-de-arquivo__area-de-envio--ativa': zonaAtiva }"
-      @dragenter="zonaAtiva = true"
-      @dragleave="zonaAtiva = false"
-      @drop.stop.prevent="enviarArquivo"
-      @dragover.stop.prevent="($ev) => { $ev.dataTransfer.dropEffect = 'copy' }"
-    >
-      <template v-if="!zonaAtiva">
-        <template v-if="!model">
-          <svg
-            width="20"
-            height="20"
-          >
-            <use xlink:href="#i_+" />
-          </svg>
-          <span class="f1">
-            <slot>
-              Adicionar (ou soltar) arquivo <template
-                v-if="props.accept"
-              >({{ props.accept }})</template>
-            </slot><template v-if="props.required">&nbsp;<span class="tvermelho">*</span></template>
-          </span>
-        </template>
-
-        <img
-          v-else-if="!ativaçãoDoArquivoPendente && éImagem"
-          :src="`${baseUrl}/download/${model}?inline=true`"
-          width="100"
-          class="campo-de-arquivo__amostra ib"
-        >
-        <span
-          v-else
-          class="campo-de-arquivo__token"
-        >{{ model }}</span>
-        <button
-          v-if="model"
-          class="campo-de-arquivo__botao-de-remoção like-a__link"
-          type="button"
-          aria-label="Remover arquivo"
-          title="Remover arquivo"
-          @click="removerArquivo"
+    <template v-else>
+      <div
+        v-if="$props.tipo === 'DOCUMENTO' && !model"
```

Comment:
Não precisa acessar com `$props`
```suggestion
        v-if="tipo === 'DOCUMENTO' && !model"
```